### DISCUSSION TITLE
Recursive tables in Shapes

### DISCUSSION THREAD

**Discussion by guidonte** [electric-sql/electric#1369] [Feature requests]
I'm aware that Shapes don't support recursive tables yet - https://electric-sql.com/docs/usage/data-access/shapes#limitations-and-issues

I'd like to know whether this limitation will be addressed anytime soon or if I should consider changing my database model.

Currently, my app relies heavily on a table with several recursive relationships (it's a personal Dictionary app where Words have relationships with each other), so it would be challenging to change that cleanly.

I'd love to be able to leverage your very interesting work in my app, though.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
